src/oemof/tabular/facades/experimental/battery_electric_vehicle.py

```diff
@@ -0,0 +1,354 @@
+
+    balanced: bool = False
+
+    pkm_conversion_rate: float = 1
+
+    efficiency_mob_g2v: float = 1
+
+    efficiency_mob_v2g: float = 1
+
+    efficiency_mob_electrical: float = 1
+
+    efficiency_sto_in: float = 1
+
+    efficiency_sto_out: float = 1
+
+    def build_solph_components(self):
+        # use label as prefix for subnodes
+        self.facade_label = self.label
+        self.label = self.label + "-storage"
+
+        # convert to solph sequences
+        self.availability = solph_sequence(self.availability)
+        self.max_charging_power = solph_sequence(self.max_charging_power)
+
+        # TODO: check if this is correct
+        self.nominal_storage_capacity = self.storage_capacity
+        # self.nominal_storage_capacity = self._nominal_value(
+        #     self.storage_capacity)
+
+        self.balanced = self.balanced  # TODO to be false in multi-period
+
+        # create internal bus
+        internal_bus = Bus(label=self.facade_label + "-bus")
+        self.bus = internal_bus
+        subnodes = [internal_bus]
+
+        # ##### Vehicle2Grid Converter #####
+        if self.v2g:
+            vehicle_to_grid = Converter(
+                label=self.facade_label + "-v2g",
+                inputs={
+                    internal_bus: Flow(
+                        # variable_costs=self.carrier_cost,
+                        # **self.input_parameters
+                    )
+                },
+                outputs={
+                    self.electricity_bus: Flow(
+                        nominal_value=self._nominal_value(
+                            value=self.max_charging_power
+                        ),
+                        # max=self.availability, # doesn't work with investment
+                        variable_costs=None,
+                        investment=self._investment(bev=True),
+                    )
+                },
+                # Includes storage charging efficiencies
+                conversion_factors={internal_bus: self.efficiency_mob_g2v},
+                # TODO check efficiencies
+            )
+            subnodes.append(vehicle_to_grid)
+
+        # ##### Charging Converter #####
+
+        # grid_to_vehicle = Converter(
+        #     label=self.facade_label + "-g2v",
+        #     inputs={
+        #         self.electricity_bus: Flow(
+        #             # **self.output_parameters
+        #         )
+        #     },
+        #     outputs={internal_bus: Flow(
+        #         nominal_value=self._nominal_value(
+        #             value=self.max_charging_power),
+        #         max=self.availability,
+        #         variable_costs=None,
+        #         investment=self._investment(bev=True),
+        #     )},
+        #     conversion_factors={
+        #     self.electricity_bus: self.efficiency_charging},
+        #     # TODO maybe add battery efficiency + charger efficiency
+        # )
+        # subnodes.append(grid_to_vehicle)
```

Comment:
you should remove commented-out code

Reply:
I just kept this, as I might have to go back to this version. But will remove it, as soon as I know.